nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/main/java/org/apache/nifi/processors/attributes/UpdateAttribute.java

```diff
@@ -259,6 +264,14 @@ public void onScheduled(final ProcessContext context) throws IOException {
 
             context.getStateManager().setState(tempMap, Scope.LOCAL);
         }
+
+        defaultActions = getDefaultActions(context.getProperties());
```

Comment:
This is probably a crazy edge case, but what if the regex for Delete Attributes Expression is defined by [Expression Language using flow file attributes](https://issues.apache.org/jira/browse/NIFI-1123)?